New semantic space

minimalparts opened the pull request:
Ok. I struggled a bit with GitHub on this one, so I hope I didn't do anything wrong... The new semantic space is here: aurelieherbelot.net/pears-demo/wikiwoods.dump.bz2. I wasn't sure where/how to include instructions about downloading/uncompressing it. Please advise!

Thanks @nandajavarma for the great scripts! :-)

minimalparts commented:
Damn, I did do something wrong. It looks like I somehow reverted some import statements to a previous version. Sorry and help! :-(

nandajavarma commented:
Happens happens. :)
Can you share you git log output? And which all files are you making the changes in this PR?

minimalparts commented:
Hey, thanks! Which type of log do you need? I was changing views.py, findBestPears.py, scorePages.py and shared_pears_ids.txt. Plus deleting the old wikiwoods db.

What happened was that I'd started on this branch when I had to checkout your last PR. I stashed my work, but then didn't recover it properly. So I just deleted my branch again, checked out development anew, and started again from scratch with a new branch bearing the same name as the one I'd deleted...

minimalparts committed:
New semantic space - take 2

minimalparts committed:
New semantic space - take 2

minimalparts committed:
New semantic space - take 2

minimalparts commented:
Alright. So here is take 2. I have made the following changes:
- Replaced posTagger.py with lemmatise.py (no need for tagger anymore at query stage).
- Changed views.py to call lemmatise.py.
- Changed mkQueryDist in utils.py.
- Changed thresholds in scorePages.py.
- Updated shared_pears_ids.txt.
- Uncommented the line that outputs similarity scores for pears in findBestPears.py, just for troubleshooting.
